### DISCUSSION TITLE
[AnalyticalTable] Wrapping

### DISCUSSION THREAD

**Discussion by DrMalchev** [UI5/webcomponents-react#6162] [General]
Fiori guidelines clearly states that wrapping in analytical table is not desired. However we have a request from a customer to investigate if we can do this in order to enhance his flow. As this is not directly supported, I tried using cell and then using the Text component with wrapping. The issue is that the row in the table does not update its height based on the new cell height.

**Comment by Lukas742** [comment]
Hi @DrMalchev 

the AnalyticalTable doesn't support dynamic row heights by design. It is only possible adjusting the heights for all rows (`rowHeight` prop) or the height of the header row (`headerRowHeight` prop).
